build/config/PublishData.json

```diff
@@ -1,18 +1,18 @@
 {
     "branches": {
-        "dev15.5.x": {
+        "master-vs-deps": {
             "nugetKind": "PerBuildPreRelease",
-            "version": "2.6.*",
+            "version": "2.7.*",
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev15.5", "dev15.5p3" ]
+            "channels": [ "dev15.6" ]
```

Comment:
Just curious, what does the `channel` do?

Reply:
This updates value in the dotnet/versions repo.